doc/source/release-notes/1.0.4.rst

```diff
@@ -0,0 +1,98 @@
+.. _1.0.4:
+
+=======================================
+Percona Monitoring and Management 1.0.4
+=======================================
+
+:Date: September 13, 2016
+:PMM Server: https://hub.docker.com/r/percona/pmm-server/
+:PMM Client: https://www.percona.com/downloads/TESTING/pmm/pmm-client.tar.gz
+
+Upgrading
+=========
+
+1. :ref:`Remove all PMM Clients <remove-client>`
+
+#. :ref:`Remove PMM Server <remove-server>`
+
+#. :ref:`Create and run the PMM Server container <server-container>`            
+
+#. :ref:`Install PMM Clients <client-install>` on all your monitored hosts 
+
+Changes
+=======
+
+**PMM Server changes:**
+
+* Grafana 3.1.1
+
+* Prometheus 1.0.2
+
+* Added SSL and HTTP password support
+  for Prometheus, Consul API, and QAN app
+
+* Removed the extra ``alias`` label for Prometheus
+
+* Added ``MongoDB RocksDB``, ``PXC/Galera Cluster Overview`` dashboards
+
+* Intriduced some visual amendments to the dashboards
+
+* Added ability to save predefined dashboards in place
+
+* Query Analytics App:
+  * Added sparkline charts to metrics
+  * Added search by query fingerprint
+  * Various smaller fixes and improvements
+
+**PMM Client changes:**
+
+* Renamed services managed by ``pmm-admin``:
+  * ``os`` > ``linux:metrics``
+  * ``mysql`` > ``mysql:metrics``
+  * ``queries`` > ``mysql:queries``
+  * ``mongodb`` > ``mongodb:metrics``
+
+* Added group commands:
+  * ``pmm-admin add mysql`` and ``pmm-admin rm mysql``:
+    add and remove ``linux:metrics``, ``mysql:metrics``,
+    and ``mysql:queries`` services
+  * ``pmm-admin add mongodb`` and ``pmm-admin rm mongodb``:
+    add and remove ``linux:metrics`` and ``mongodb:metrics`` services
+
+* Added SSL and HTTP password support for *PMM Server*,
```

Comment:
Update to this, let's say differently:
`* Added options to support SSL and HTTP password protection on PMM server.`